tests/tests.toml

```diff
@@ -487,6 +487,10 @@ asr = true
 filename = "../integration_tests/global_syms_01.py"
 asr = true
 
+[[test]]
+filename = "../integration_tests/test_list_const.py"
```

Comment:
Minor: I think this should be named `test_const_list` since it is test for `const List` and not `List Constant`.